docs/README.md

```diff
@@ -215,356 +153,385 @@ for these executables in your `$PATH`.
 +--------+------------------------+----------------------------------------------------------------------------+
 ```
 
+#### 2.2 Creating a new index
+
+Now that we have our development environment setup, the next step is to create an index.
+
+```bash
+forc index new hello-index --namespace my_project && cd hello-index
```

Comment:
@deekerno 
- So the example is called "Hello Index", but if you check the default code that gets created with `forc index new`, it's just indexing blocks and transactions. 
- So that's why I mention "visit block explorer for more info".
- Let me know if it still doesn't make sense